src/vs/workbench/api/common/extHostTerminalShellIntegration.ts

```diff
@@ -160,30 +171,42 @@ class InternalTerminalShellIntegration extends Disposable {
 			get cwd(): URI | undefined {
 				return that._cwd;
 			},
-			executeCommand(commandLine): vscode.TerminalShellExecution {
-				that._onDidRequestShellExecution.fire(commandLine);
+			// executeCommand(commandLine: string): vscode.TerminalShellExecution;
+			// executeCommand(executable: string, args: string[]): vscode.TerminalShellExecution;
```

Comment:
rm?

Reply:
I left that in intentionally as those are the actual apis. I'd uncomment but TS doesn't like overloads